pkg/store/cache/caching_bucket.go

```diff
@@ -26,92 +29,178 @@ import (
 )
 
 const (
+	config = "config"
```

Comment:
I think this complicates reading the code because the reader woudln't expect the `config` variable contains the `"config"` label name. I would keep it simpler and just remove this var.